step.sh

```diff
@@ -1,9 +1,27 @@
 #! /bin/bash
-set -euo pipefail
+set -eo pipefail
```

Comment:
`-u` makes the script fail at an undefined variable, and `BITRISEIO_BUILD_CACHE_ENABLED` could be undefined.

Reply:
TIL what the `u` flag means.